app/controllers/transactions_controller.rb

```diff
@@ -0,0 +1,36 @@
+class TransactionsController < ApplicationController
+
+  customer_tab :all
+  before_action :authenticate_user!
+  before_action :check_acting_as
+
+  include TransactionSearch
+
+  def initialize
+    @active_tab = "accounts"
+    super
+  end
+
+  def index_with_search
```

Comment:
My first thought was to consider changing the URL if and when we decide to have separate all-transactions and in-review views. But considering this the the URL we're going to be emailing, it would make sense to keep it stable. Let's go with `/transactions/in_review` now.